apps/api/src/controllers/request-helpers.ts

```diff
@@ -86,15 +87,29 @@ export function ensurePaging(req: Request) {
   return data;
 }
 
+/**
+ * Ensure the query parameters are valid agent competition params (unified paging and filters)
+ * @param req Express request
+ * @returns The agent competition filters
+ */
+export function ensureAgentCompetitionParams(req: Request) {
```

Comment:
made some small changes so that the rpc could separate the filtering args from the paging args